docs/constants/navigation.js

```diff
@@ -193,6 +193,7 @@ const general = [
       makePage('router/reference/troubleshooting.mdx'),
       makePage('router/reference/faq.mdx'),
     ]),
+    makeGroup('Migrate', [makePage('router/migrate/from-react-navigation.mdx')]),
```

Comment:
```suggestion
    makeGroup('Migration', [makePage('router/migrate/from-react-navigation.mdx')]),
```

Let's change this to Migration.